docroot/modules/custom/foia_webform/tests/src/Kernel/FoiaSubmissionServiceApiTest.php

```diff
@@ -201,11 +201,17 @@ public function testAssesmbleRequestData() {
     $webform = Webform::create([
       'id' => $this->randomMachineName(),
     ]);
-    $webform->set('foia_template', 1);
+    $webform->set('foia_template', [
+      '#type' => 'checkbox',
+      '#title' => t("Use FOIA Agency template"),
+      '#disabled' => TRUE,
+      '#default_value' => 'foia_template',
+      '#value' => 'foia_template',
+    ]);
```

Comment:
`foia_template` is a boolean value, so the existing code should be setting it properly. I think there's an issue preventing it from working, but I just checked with this approach and it still doesn't work, so I'd leave this code out.